RELEASE_NOTES.md

```diff
@@ -503,6 +503,10 @@ The following is a summary of the main features added to darktable
   version, this can be supported by reverting the following change:
   remove line 241 of darktable.css file on your system. See:
   https://github.com/darktable-org/darktable/issues/13166
+  
+- Beginning with this release a new support policy regarding macOS versions will be in place.
+  darktable releases will just support macOS versions that are also supported by apple. 
+  So release 4.4 will drop support for macOS versions older than 11.3.  
```

Comment:
So release 4.4 is dropping...

As this will be used at the time we release 4.4. Also don't we spell it MacOS instead of macOS?

Reply:
> Also don't we spell it MacOS instead of macOS?

The correct spelling is macOS since 2016. BTW, @MStraeten, we shouldn't write apple in lowercase, it's a trademark and company name, so it should keep the original spelling even if we have a different style.

Reply:
Updated …